data/transactions/logic/eval_test.go

```diff
@@ -4134,6 +4134,22 @@ func TestDig(t *testing.T) {
 	testPanics(t, "int 3; int 2; int 1; dig 11; int 2; ==; return", 3)
 }
 
+func TestCover(t *testing.T) {
```

Comment:
It would be nice to see test cases here that check that other elements are shifted correctly

Reply:
will add these